src/CreativeCollab.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import TextEditor from './TextEditor';
+import './CreativeCollab.css';
+
+class CreativeCollab extends React.Component {
+    constructor(){
+        super();
+        this.state = {
+            story: ''
+        };
+
+        this.addTextFromFirstEditor = this.addTextFromFirstEditor.bind(this);
+        this.addTextFromSecondEditor = this.addTextFromSecondEditor.bind(this);
+    }
+
+    addTextFromFirstEditor(){
```

Comment:
There's some duplicate logic here with `addTextFromFirstEditor ` and `addTextFromSecondEditor`. Let's refactor that in a future PR 🙌🏽

Reply:
Agree. 
I've just refactored it, check the new commit

Reply:
👏